### PR TITLE
Feature/spring boot 2.x accumulo 2.1

### PR DESCRIPTION
**Pull request by jwomeara**

### CODE REVIEW

**Review by matthpeterson — DISMISSED** [review]

**Review by matthpeterson — DISMISSED** [review]

**Review by apmoriarty — DISMISSED** [review]

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -18,10 +18,10 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v1
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
     - uses: actions/cache@v1
```
Comment (nonessentialprototype) [inline .github/workflows/tests.yaml]: Might want to consider updating checkouts, setup-java, and cache to v3 since this stuff is already getting updated.

**Review by nonessentialprototype — DISMISSED** [review]

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (jwomeara) [reply .github/workflows/tests.yaml]: didn't you work an effort to update that across the board for all of our projects?  any reason this one was left out?  I imagine we want this on main as well, right?

Comment (nonessentialprototype) [reply .github/workflows/tests.yaml]: I have started PR's on each microservice to update these.  DataWave mono repo has already been updated and merged.

Comment (jwomeara) [reply .github/workflows/tests.yaml]: If you're still working that effort then I'll be happy to merge in your changes for this repo once you make them.

**Review by matthpeterson — APPROVED** [review]

**Pull request merged**